ccd-definition/AuthorisationCaseField/CareSupervision/cafcass-solicitor.json

```diff
@@ -461,6 +461,13 @@
     "UserRole": "[CAFCASSSOLICITOR]",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "courtBundleListV2",
+    "UserRole": "[CAFCASSSOLICITOR]",
```

Comment:
Maybe just double check CAFCASS don't need to be able to see the other documents

Reply:
already added "RU" to "caseworker-publiclaw-cafcass". Think it should be enough for cafcass to view it.